pkg/api/vm/formatter.go

```diff
@@ -179,7 +179,7 @@ func (vf *vmformatter) canStop(vm *kubevirtv1.VirtualMachine, vmi *kubevirtv1.Vi
 			return false
 		case kubevirtv1.RunStrategyManual, kubevirtv1.RunStrategyRerunOnFailure:
 			// need to check VMI status
-			if vmi == nil || vmi.Status.Phase != kubevirtv1.Running {
+			if vmi == nil {
```

Comment:
So if VMI is existing, harvester will allow to force stop VM to remove VMI?
Is it the purpose?

Reply:
Yes, users should be able to retry the `stop` action as long as the VM is not stopped.

Reply:
Ok LGTM.